osbuild/tmpfs.py

```diff
@@ -0,0 +1,36 @@
+
+import os
+import subprocess
+import tempfile
+
+
+__all__ = [
+    "TmpFs",
+]
```

Comment:
We're not using this module at all. Should we drop it? (Can be done in a follow-up, of course)

Reply:
I was using it in a different WIP PR, so didn't want to drop it in the same commit as the split, to make it easier to revert if we want it again. But yeah, if I don't end up filing that PR we might as well drop this.